Add `lastUpdated` global and frontmatter options (#256)
withastro/starlight: docs/src/content/docs/reference/frontmatter.md

  }>;
}
```

### `lastUpdated`

**type:** `Date | boolean`

> Overrides either the [global `lastUpdated` option](/reference/configuration/#lastupdated) or the last update date of the current page generated from the Git history. If a date is specified, it should be a valid [YAML timestamp](https://yaml.org/type/timestamp.html).

Reviewer: ```suggestion
Overrides the [global `lastUpdated` option](/reference/configuration/#lastupdated). If a date is specified, it must be a valid [YAML timestamp](https://yaml.org/type/timestamp.html) and will override the date stored in Git history.
```

  }>;
}
```

### `lastUpdated`

> **type:** `Date`

Reviewer: I’d suggest allowing us to set an override here either with a `Date` (for an explicit override to a specific value) or a `boolean` (to enable/disable the default git-based date for specific pages). This is also how the frontmatter [`editUrl`](https://starlight.astro.build/reference/frontmatter/#editurl) option works.

```suggestion
**type:** `Date | boolean`
```

The docs below would also need a small tweak to reflect that.
